pkg/cmd/experimental/diagnostics/options/diagnostics.go

```diff
@@ -1,46 +1,36 @@
 package options
 
 import (
-	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
-	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 	"github.com/spf13/pflag"
-	"io"
-)
 
-// all of the diagnostics commands will bind these options
-type DiagnosticsOptions struct {
-	Diagnostics *util.StringList // named diagnostics to run
-	DiagLevel   int              // show output of this priority or higher
-	DiagFormat  string           // format of output - text/json/yaml
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 
-	Output cmdutil.Output // this is used for discovery and diagnostic output
+	"github.com/openshift/origin/pkg/diagnostics/log"
+)
+
+type RecommendedLoggerOptionsFlags struct {
+	Level  FlagInfo
+	Format FlagInfo
 }
 
-func NewDiagnosticsOptions(out io.Writer) *DiagnosticsOptions {
-	return &DiagnosticsOptions{
-		Diagnostics: &util.StringList{}, // have to instantiate in order to bind flag
-		Output:      cmdutil.Output{out},
+// default overrideable flag specifications to be bound to options.
+func RecommendedLoggerOptionFlags() RecommendedLoggerOptionsFlags {
```

Comment:
"Recommended" just always struck me as odd. Recommended by whom, for what? And... a whole type just for the recommendation?

"New" or "Standard"LoggerOptionFlags would make sense to me, being in the context of this package, you pretty much know what it's for, and feel free to reuse/override as appropriate.

I have a feeling you've had this conversation before however...